src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7663,16 +7663,7 @@ private static void CombineExtensionMethodArguments(BoundExpression receiver, An
             ImmutableArray<TypeWithAnnotations> typeArgumentsWithAnnotations,
             BindingDiagnosticBag diagnostics)
         {
-            int arity;
-            if (typeArgumentsWithAnnotations.IsDefault)
-            {
-                arity = 0;
-            }
-            else
-            {
-                arity = typeArgumentsWithAnnotations.Length;
-            }
-
+            int arity = typeArgumentsWithAnnotations.IsDefault ? 0 : typeArgumentsWithAnnotations.Length;
```

Comment:
>int arity = typeArgumentsWithAnnotations.IsDefault ? 0 : typeArgumentsWithAnnotations.Length; [](http://example.com/codeflow?start=12&length=93)

Is this a style change? Consider reverting. I think our general policy that style changes are not welcome

Reply:
It is, but it's local/contained and I think it's easy to review. A small improvement doesn't hurt

Reply:
> A small improvement doesn't hurt

I, for example, do not see this change as an improvement. That is why we do not change style. Each developer has personal preferences and personal definition of improvement with respect to style.